src/compiler/emitter.ts

```diff
@@ -1850,6 +1850,12 @@ var __param = (this && this.__param) || function (paramIndex, decorator) {
                 emitListWithSpread(node.arguments, /*multiLine*/ false, /*trailingComma*/ false);
                 write(")");
             }
+            
+            function emitBracedEnclosedCommaList(nodes: Node[]) {

```

Comment:
These are parens, not braces. But I don't think you need a separate function for this.